### ISSUE TITLE
`onepub doctor` crashes in windows

### ISSUE THREAD

**Issue by ahmednfwela** [onepub-dev/onepub#193]
running `onepub doctor` on my machine:
```
Platform
Dart version:   2.18.4
Dart path:      C:\tools\flutter\bin\cache\dart-sdk

OS:             windows
OS version:     "Windows 10 Pro N" 10.0 (Build 1904

Locale:         en-US
Path separator: \


URLs
Web site:       https://onepub.dev/
API endpoint:   https://onepub.dev/api/snssdavuhn/

Environment
Pub Cache:      C:\Users\xxx\AppData\Local\Pub\Ca
PUB_CACHE:      not set and not used.
PATH:
                C:\Program Files\PowerShell\7
                C:\Program Files (x86)\VMware\VMwar
                C:\PROGRA~2\VMware\VMWARE~1\bin
                C:\tools\ruby31\bin
                C:\PROGRA~1\ECLIPS~1\JRE-18~1.10-\b
                C:\PROGRA~1\ECLIPS~1\JRE-17~1.12-\b
                C:\PROGRA~1\MIE74D~1\JDK-11~1.7-H\b
                C:\PROGRA~2\COMMON~1\Intel\SHARED~1
                C:\tools\ruby27\bin
                C:\PROGRA~1\ADOPTO~1\JRE-15~1.36-\b
                C:\PROGRA~1\ADOPTO~1\JRE-14~1.12-\b
                C:\Windows\system32
                C:\Windows
                C:\Windows\System32\Wbem
                C:\Windows\System32\WindowsPowerShe
                C:\Windows\System32\OpenSSH
                C:\PROGRA~1\NVIDIA~1\NVIDIA~2
                C:\PROGRA~1\MICROS~1\bin
                C:\PROGRA~1\MICROS~2\130\Tools\Binn
                C:\PROGRA~1\MICROS~2\CLIENT~1\ODBC\
                C:\PROGRA~3\CHOCOL~1\bin
                C:\PROGRA~3\nvm
                C:\PROGRA~1\OPENSS~1
                C:\PROGRA~1\Android\ANDROI~1\jre\bi
                C:\PROGRA~2\NVIDIA~1\PhysX\Common
                C:\PROGRA~2\Google\CLOUDS~1\GOOGLE~
                C:\PROGRA~1\PuTTY
                C:\PROGRA~1\MIE74D~1\WEBPLA~1
                C:\PROGRA~2\MICROS~3\150\DTS\Binn
                C:\PROGRA~1\RedHat\JAVA-1~1.2-1\MIS
                C:\PROGRA~1\RedHat\JAVA-1~1.2-1\bin
                C:\PROGRA~2\MICROS~3\150\Tools\Binn
                C:\PROGRA~1\MICROS~2\150\Tools\Binn
                C:\PROGRA~1\MICROS~2\150\DTS\Binn
                C:\PROGRA~2\Yarn\bin
                C:\PROGRA~1\LINQPad6
                C:\ProgramData\chocolatey\lib\maven
                C:\PROGRA~1\MICROS~2\140\Tools\Binn
                C:\Users\xxx\AppData\Local\Androi
                C:\PROGRA~1\dotnet
                C:\PROGRA~1\Git\cmd
                C:\PROGRA~3\CHOCOL~1\lib\maven\APAC
                C:\PROGRA~1\nodejs
                C:\PROGRA~1\TORTOI~1\bin
                C:\PROGRA~1\Go\bin
                E:\LINGO64_19
                C:\PROGRA~1\nodejs
                C:\PROGRA~1\Docker\Docker\RESOUR~1\
                C:\PROGRA~1\POWERS~1\7
                C:\PROGRA~2\COMMON~1\Oracle\Java\ja
                C:\Program Files\Microsoft VS Code\
                C:\Program Files\dotnet
                C:\Program Files\Git\cmd
                C:\Program Files\Docker\Docker\reso
                C:\Program Files\PowerShell\7       
                C:\Program Files (x86)\Google\Cloud
                C:\Program Files (x86)\Google\Cloud
                C:\Program Files\MySQL\MySQL Shell
                C:\Users\xxx\AppData\Local\MICROS
                C:\Users\xxx\AppData\Local\GITHUB
                C:\Users\xxx\DOTNET~1\tools
                C:\PROGRA~1\MINGW-~1\X86_64~1.0-P\m
                C:\Users\xxx\AppData\Local\Micros
                C:\Users\xxx\.dotnet\tools
                C:\tools\flutter\bin\cache\dart-sdk
                C:\tools\flutter\bin
                C:\Users\xxx\AppData\Local\Pub\Ca
                C:\tools\dart-sdk\bin
                C:\Users\xxx\AppData\Local\Progra
                C:\PROGRA~1\MongoDB\Tools\100\bin
                C:\Users\xxx\AppData\Local\Yarn\b
                C:\PROGRA~1\MongoDB\Server\4.4\bin
                C:\PROGRA~2\INNOSE~1
                C:\ProgramData\nvm
                C:\Program Files\nodejs
                C:\Users\xxx\go\bin
                C:\<HOME>\npm
                C:\Gradle\GRADLE~1.3\bin
                C:\Users\xxx\AppData\Local\Progra
                C:\tools\bin
Invalid argument(s): path must not be empty.
#0      _Is.exists (package:dcli_core/src/functions/is.dart:109:7)
#1      exists (package:dcli_core/src/functions/is.dart:48:11)
#2      exists (package:dcli/src/functions/is.dart:56:10)
#3      DoctorCommand._printPATH (package:onepub/src/commands/doctor.dart:69:12)
#4      DoctorCommand._printEnvironment (package:onepub/src/commands/doctor.dart:53:5)
#5      DoctorCommand.run (package:onepub/src/commands/doctor.dart:33:5)
#6      CommandRunner.runCommand (package:args/command_runner.dart:209:27)
#7      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:119:25)
#8      new Future.sync (dart:async/future.dart:302:31)
#9      CommandRunner.run (package:args/command_runner.dart:119:14)
#10     entrypoint (package:onepub/src/entry_point.dart:38:20)
#11     main (file:///C:/Users/xxx/AppData/Local/Pub/Cache/git/onepub-7a3416824f0b35a82f1456e26af4bff5a335ae28/bin/onepub.dart:11:9)
#12     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
#13     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```

**Comment by bsutton** [comment]
You have an empty path on your PATH.

I will update doctor you skip empty paths.

**Comment by bsutton** [comment]
This has been fixed and will be released tomorrow as 2.0.5.
